docs/01-Using-Fleet/FAQ.md

```diff
@@ -177,4 +178,8 @@ Invalid interpolation format for "fleet01" option in service "services": "fleetd
 Failed to run docker-compose
 ```
 
-You are probably running an old version of Docker. You should download the installer for your platform from https://docs.docker.com/compose/install/
\ No newline at end of file
+You are probably running an old version of Docker. You should download the installer for your platform from https://docs.docker.com/compose/install/
+
+## How does Fleet deal with IP duplication?
+
+Fleet relies on UUIDs so any overlap with host IP addresses should not cause a problem. The only time this might be an issue is if you are running a query that involves a specific IP address that exists in multiple locations as it might return multiple results - Fleet's teams feature can be used to restrict queries to specific hosts.
```

Comment:
Let's light up the relevant words with a link to the docs:
https://fleetdm.com/docs/using-fleet/teams

If we pick semantic words, e.g.
```
[Fleet's teams feature](…) can be
```

then it'll help people using search engines be able to find the documentation page more easily, even from unrelated searches.  (i.e. helps SEO)

Reply:
Added a commit just now 👍🏻